src/EmployerAccounts/SFA.DAS.EmployerAccounts/Application/Queries/GetEmployerAccountTaskList/GetEmployerAccountTaskListQuery.cs

```diff
@@ -1,10 +1,13 @@
-﻿using MediatR;
+﻿using System.Collections.Generic;
+using MediatR;
+using SFA.DAS.SharedOuterApi.Models.ProviderRelationships;
 
 namespace SFA.DAS.EmployerAccounts.Application.Queries.GetEmployerAccountTaskList
 {
     public class GetEmployerAccountTaskListQuery : IRequest<GetEmployerAccountTaskListQueryResult>
     {
         public string HashedAccountId { get; set; }
         public long AccountId { get; set; }
+        public List<Operation> Operations { get; set; } = new List<Operation> { Operation.Recruitment, Operation.RecruitmentRequiresReview };
     }
```

Comment:
I would be tentative about having preset values in a class declaration unless the operations are to never change. Removes hidden logic and in the case of those values never changing it might be worth while using a private setter.

Reply:
Fixed